homeassistant/components/asuswrt/bridge.py

```diff
@@ -281,7 +288,7 @@ async def _get_label_mac(self) -> None:
         """Get label mac information."""
         label_mac = await self._get_nvram_info("LABEL_MAC")
         if label_mac and "label_mac" in label_mac:
-            self._label_mac = format_mac(label_mac["label_mac"])
+            self._label_mac: str | None = format_mac(label_mac["label_mac"])
```

Comment:
This should not be typed here, as this is a result from the use of `format_mac`
```suggestion
            self._label_mac = format_mac(label_mac["label_mac"])
```